docs/src/components/sidebarIndex.js

```diff
@@ -31,7 +31,7 @@ const sidebarIndex: Array<sidebarIndexType> = [
   },
   {
     sectionName: 'Foundation',
-    pages: ['Heading', 'Icon', 'Text'],
+    pages: ['Color', 'Heading', 'Icon', 'Text'],
```

Comment:
This seems like an odd section choice to me, since Color isn't a component. Would "Guidelines" be a better fit?